### ISSUE TITLE
Issue installing redis

### ISSUE THREAD

**Issue by thomaspicquet** [FriendsOfPHP/pickle#190]
Hi,

I'm getting this errors when installing redis `pickle install redis --defaults`
```
In VersionParser.php line 176:

  Invalid version string "5.3.2RC1-alpha"
```

I haven't found a way to update which version it installs so I'm not sure how to fix it

thanks

**Comment by pierrejoye** [comment]
Hello,

thanks for the report!

pls use master, i wil a work around for extensions not using semver.

**Comment by thomaspicquet** [comment]
Thanks @pierrejoye , I will

**Comment by mlocati** [comment]
This issue should have been fixed by #201
